packages/babel-check-binary-expressions/src/index.js

```diff
@@ -0,0 +1,8 @@
+export default function(left, right) {
+  // eslint-disable-next-line no-undef
+  if (left instanceof BigInt && right instanceof BigInt) {
```

Comment:
Yes sure, I was thinking about how it would be bundled but that's an unnecessary change anyway.